### PR TITLE
refactor(tests): make error consistent

### PR DESCRIPTION
**Pull request by mannyrivera2010**
AMLNG-728

**Description**     
Errors have same Consistent Format for all endpoints

**Acceptance Criteria**   
Make sure that all endpoints have the correct error format
 
**How to test code**    
```
make ptest
```
Validate that all tests passes

**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
- [x] Read CONTRIBUTING.md
- [x] Write code to complete task 
- [x] Python Code is PEP8 Compliant
- [x] Add unit tests for new code
- [x] All tests must pass before merging the pull request
- [ ] At least 2 people reviewed code

### CODE REVIEW

File: ozpcenter/errors.py [hunk 1]
```diff
@@ -55,10 +67,10 @@ def exception_handler(exc, context):
     By default we handle the REST framework `APIException`, and also
     Django's built-in `Http404` and `PermissionDenied` exceptions.
     Any unhandled exceptions may return `None`, which will cause a 500 error
-    to be raised.
+    to be raised.return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
```
Comment (blynch11p) [inline ozpcenter/errors.py]: Looks like copy/paste errors in here.

File: ozpcenter/errors.py [hunk 2]
```diff
@@ -55,10 +67,10 @@ def exception_handler(exc, context):
...
     """
     request = context.get('request')
-    logger.exception(exc, extra={'request': request})
+    logger.excereturn Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)ption(exc, extra={'request': request})
```
Comment (blynch11p) [inline ozpcenter/errors.py]: and here

File: ozpcenter/errors.py [hunk 3]
```diff
@@ -74,7 +86,7 @@ def exception_handler(exc, context):
             data = {'error': True, 'detail': exc.detail, 'error_code': exc.default_code}
 
         set_rollback()
-        return Response(data, status=exc.status_code, headers=headers)
+        return Response(datreturn Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)a, status=exc.status_code, headers=headers)
```
Comment (blynch11p) [inline ozpcenter/errors.py]: and here

File: ozpcenter/api/profile/views.py [hunk 4]
```diff
@@ -192,25 +193,29 @@ def create(self, request, profile_pk=None):
...
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(POST) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise errors.NotImplemented('HTTP Verb(POST) Not Supported')
 
     def update(self, request, pk=None, profile_pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb(PUT) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(PUT) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise errors.NotImplemented('HTTP Verb(POST) Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/profile/views.py]: Should be (PUT)

File: ozpcenter/api/profile/views.py [hunk 5]
```diff
@@ -192,25 +193,29 @@ def create(self, request, profile_pk=None):
...
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(PATCH) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise errors.NotImplemented('HTTP Verb(POST) Not Supported')
 
     def destroy(self, request, pk=None, profile_pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb(DELETE) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(DELETE) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise errors.NotImplemented('HTTP Verb(POST) Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/profile/views.py]: Should be (DELETE)

File: ozpcenter/api/subscription/views.py [hunk 6]
```diff
@@ -62,7 +62,8 @@ def create(self, request):
 
         if not serializer.is_valid():
             logger.error('{0!s}'.format(serializer.errors))
-            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+            raise errors.RequestException('')
```
Comment (mannyrivera2010) [inline ozpcenter/api/subscription/views.py]: Should it be ValidationException???

File: ozpcenter/api/profile/views.py [hunk 7]
```diff
@@ -348,8 +353,9 @@ def update(self, request):
             data=request.data, context={'request': request}, partial=True)
         if not serializer.is_valid():
             logger.error('{0!s}'.format(serializer.errors))
-            return Response(serializer.errors,
-                status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors   return Response(serializer.errors,
+            #    status=status.HTTP_400_BAD_REQUEST)
+            raise errors.RequestException('{0}'.format(serializer.errors))
```
Comment (mannyrivera2010) [inline ozpcenter/api/profile/views.py]: Should it be ValidationException???

File: ozpcenter/api/listing/views.py [hunk 8]
```diff
@@ -1128,28 +1138,33 @@ def create(self, request):
...
+        # ConsistentErrors  return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
 
     def destroy(self, request, pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors  return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/listing/views.py]: should be (errors.NotImplemented)

File: ozpcenter/api/listing/views.py [hunk 9]
```diff
@@ -1128,28 +1138,33 @@ def create(self, request):
...
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
 
     def partial_update(self, request, pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors  return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/listing/views.py]: should be (errors.NotImplemented)

File: ozpcenter/api/listing/views.py [hunk 10]
```diff
@@ -1128,28 +1138,33 @@ def create(self, request):
...
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
 
     def update(self, request, pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/listing/views.py]: should be (errors.NotImplemented)

File: ozpcenter/api/listing/views.py [hunk 11]
```diff
@@ -1128,28 +1138,33 @@ def create(self, request):
...
 
     def retrieve(self, request, pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise NotImplemented('HTTP Verb Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/listing/views.py]: should be (errors.NotImplemented)

File: ozpcenter/api/listing/views.py [hunk 12]
```diff
@@ -1128,28 +1138,33 @@ def create(self, request):
...
```
Comment (mannyrivera2010) [inline ozpcenter/api/listing/views.py]: should be (errors.NotImplemented)

Comment (mannyrivera2010) [reply ozpcenter/api/listing/views.py]: should have unittest for this call

File: ozpcenter/api/library/views.py [hunk 13]
```diff
@@ -236,6 +240,8 @@ def update_all(self, request):
         current_request_username = request.user.username
         errors, data = model_access.batch_update_user_library_entry(current_request_username, request.data)
         if errors:
-            return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors  return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
+            raise exc_errors.RequestException('{0}'.format(errors))
```
Comment (mannyrivera2010) [inline ozpcenter/api/library/views.py]: `exc_errors` should be `errors`

File: ozpcenter/api/library/views.py [hunk 14]
```diff
@@ -171,7 +174,8 @@ def import_bookmarks(self, request):
 
         errors, data = model_access.import_bookmarks(current_request_username, bookmark_notification_id)
         if errors:
-            return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors  return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
+            raise exc_errors.RequestException('{0}'.format(errors))
```
Comment (mannyrivera2010) [inline ozpcenter/api/library/views.py]: `exc_errors` should be `errors`

File: ozpcenter/api/library/views.py [hunk 15]
```diff
@@ -101,8 +102,10 @@ def create(self, request):
 
         if not serializer.is_valid():
             logger.error('{0!s}'.format(serializer.errors))
-            return Response(serializer.errors,
-                status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors   return Response(serializer.errors,
+            #                           status=status.HTTP_400_BAD_REQUEST)
+            raise exc_errors.RequestException('{0}'.format(serializer.errors))
```
Comment (mannyrivera2010) [inline ozpcenter/api/library/views.py]: `exc_errors` should be `errors`

File: ozpcenter/api/library/views.py [hunk 16]
```diff
@@ -16,6 +16,7 @@
 from rest_framework.response import Response
 
 from ozpcenter import permissions
+from ozpcenter import errors as exc_errors
```
Comment (mannyrivera2010) [inline ozpcenter/api/library/views.py]: `exc_errors` should be `errors`

File: ozpcenter/api/image/views.py [hunk 17]
```diff
@@ -122,8 +122,10 @@ def create(self, request):
             context={'request': request})
         if not serializer.is_valid():
             logger.error('{0!s}'.format(serializer.errors))
-            return Response(serializer.errors,
-                status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors  return Response(serializer.errors,
+            #                    status=status.HTTP_400_BAD_REQUEST)
+            raise errors.RequestException('{0}'.format(serializer.errors))
```
Comment (mannyrivera2010) [inline ozpcenter/api/image/views.py]: should be errors.ValidationException

File: ozpcenter/api/profile/views.py [hunk 18]
```diff
@@ -192,25 +193,29 @@ def create(self, request, profile_pk=None):
...
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(PUT) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise errors.NotImplemented('HTTP Verb(POST) Not Supported')
 
     def partial_update(self, request, pk=None, profile_pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb(PATCH) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(PATCH) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        raise errors.NotImplemented('HTTP Verb(POST) Not Supported')
```
Comment (mannyrivera2010) [inline ozpcenter/api/profile/views.py]: Should be (PATCH)

File: ozpcenter/api/profile/views.py [hunk 19]
```diff
@@ -192,25 +193,29 @@ def create(self, request, profile_pk=None):
         """
         This method is not supported
         """
-        return Response({'detail': 'HTTP Verb(POST) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
+        # ConsistentErrors   return Response({'detail': 'HTTP Verb(POST) Not Supported'}, status=status.HTTP_501_NOT_IMPLEMENTED)
```
Comment (mannyrivera2010) [inline ozpcenter/api/profile/views.py]: Once Validated, remove comment

File: ozpcenter/api/library/views.py [hunk 20]
```diff
@@ -205,7 +209,7 @@ def update_all(self, request):
             works on a single instance, hence the use of a special update_all
             method. Serializers must be customized to support nested writable
             representations, and even after doing so, the input data validation
-            didn't seem acceptable. Hence this customized method
+              # ConsistentErrors didn't seem acceptable. Hence this customized method
```
Comment (mannyrivera2010) [inline ozpcenter/api/library/views.py]: Copy/Paste Error?

File: ozpcenter/api/library/views.py [hunk 21]
```diff
@@ -171,7 +174,8 @@ def import_bookmarks(self, request):
 
         errors, data = model_access.import_bookmarks(current_request_username, bookmark_notification_id)
         if errors:
-            return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
+            # ConsistentErrors  return Response({'message': '{0}'.format(errors)}, status=status.HTTP_400_BAD_REQUEST)
```
Comment (mannyrivera2010) [inline ozpcenter/api/library/views.py]: Once validated, remove comment

`Do for all comments`

File: ozpcenter/api/listing/views.py (hunk 8 above)
Comment (mannyrivera2010) [reply ozpcenter/api/listing/views.py]: make unit test for this method

File: ozpcenter/api/listing/views.py (hunk 9 above)
Comment (mannyrivera2010) [reply ozpcenter/api/listing/views.py]: make unit test for this method

File: ozpcenter/api/listing/views.py (hunk 10 above)
Comment (mannyrivera2010) [reply ozpcenter/api/listing/views.py]: make unit test for this method

File: ozpcenter/api/listing/views.py (hunk 11 above)
Comment (mannyrivera2010) [reply ozpcenter/api/listing/views.py]: make unit test for this method

File: ozpcenter/api/subscription/views.py [hunk 22]
```diff
@@ -62,7 +62,8 @@ def create(self, request):
 
         if not serializer.is_valid():
             logger.error('{0!s}'.format(serializer.errors))
-            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+            raise errors.ValidationException('')
```
Comment (mannyrivera2010) [inline ozpcenter/api/subscription/views.py]: should be `raise errors.ValidationException('{0!s}'.format(serializer.errors))`?

File: tests/ozpcenter_api/test_api_notification.py [hunk 23]
```diff
@@ -23,12 +24,13 @@
...
 
     def setUp(self):
+        pass
         """
         setUp is invoked before each test method
         """
-        self.expected_error = {'detail': 'Permission denied.',
-                               'error': True,
-                               'message': 'Only Stewards can delete notifications'}
+        # self.expected_error = {'detail': 'Permission denied.',
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 24]
```diff
@@ -374,9 +376,10 @@ def test_create_listing_notification_app_mall_steward_invalid_format(self):
         user = generic_model_access.get_profile('bigbrother').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ['Valid Listing ID is required'])
+        # ConsistentErrors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 25]
```diff
@@ -390,9 +393,11 @@ def test_create_listing_notification_app_mall_steward_invalid_id(self):
         user = generic_model_access.get_profile('bigbrother').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        # ConsistentErrors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 26]
```diff
@@ -407,9 +412,10 @@ def test_create_listing_agency_notification_app_mall_steward_invalid(self):
         user = generic_model_access.get_profile('bigbrother').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ["Notifications can only be one type. Input: ['listing', 'agency']"])
+        # ConsistentErrors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 27]
```diff
@@ -443,9 +449,10 @@ def test_create_listing_notification_org_steward_invalid_format(self):
         user = generic_model_access.get_profile('wsmith').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ["Valid Listing ID is required"])
+        # ConsistentErrors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 28]
```diff
@@ -459,9 +466,10 @@ def test_create_listing_notification_org_steward_invalid_id(self):
         user = generic_model_access.get_profile('wsmith').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ["Could not find listing"])
+        # ConsistentErrors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 29]
```diff
@@ -495,9 +503,9 @@ def test_create_agency_notification_app_mall_steward_invalid_format(self):
         user = generic_model_access.get_profile('bigbrother').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ['Valid Agency ID is required'])
+        # ConsistentErrors self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 30]
```diff
@@ -510,9 +518,9 @@ def test_create_agency_notification_app_mall_steward_invalid_id(self):
         user = generic_model_access.get_profile('bigbrother').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ['Could not find agency'])
+        # ConsistentErrors self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 31]
```diff
@@ -1047,12 +1055,20 @@ def test_delete_system_notification_apps_mall_steward(self):
     def test_delete_system_notification_org_steward(self):
         url = '/api/notification/1/'
         response = unittest_request_helper(self, url, 'DELETE', username='wsmith', status_code=403)
-        self.assertEqual(response.data, self.expected_error)
+        # replaced for consistent errors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 32]
```diff
@@ -1047,12 +1055,20 @@ def test_delete_system_notification_apps_mall_steward(self):
...
+        # self.assertEqual(response.data, self.expected_error)
+        self.assertEqual(response.data, ExceptionUnitTestHelper.permission_denied())
 
     def test_delete_system_notification_user_unauthorized(self):
         url = '/api/notification/1/'
         response = unittest_request_helper(self, url, 'DELETE', username='jones', status_code=403)
-        self.assertEqual(response.data, self.expected_error)
+        # replaced for consistent errors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_notification.py]: Remove Comment and print statements?

File: tests/ozpcenter_api/test_api_profile.py [hunk 33]
```diff
@@ -368,11 +370,10 @@ def test_update_self_for_apps_mall_steward_level_serializer_exception(self, mock
         url = '/api/self/profile/'
         data = {'id': 5, 'center_tour_flag': 4}
         response = self.client.put(url, data, format='json')
-
-        expected_data = {'center_tour_flag': ['"4" is not a valid boolean.']}
-
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(response.data, expected_data)
+        # ConsistentErrors     expected_data = {'center_tour_flag': ['"4" is not a valid boolean.']}
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_profile.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_profile.py [hunk 34]
```diff
@@ -385,9 +386,13 @@ def test_update_self_for_apps_mall_steward_level_invalid_user(self, mock_request
         self.client.login(username='invalid', password='invalid')
         response = self.client.put(url, data, format='json')
 
-        expected_data = {'detail': 'Authentication credentials were not provided.', 'error': True}
-        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
-        self.assertEqual(response.data, expected_data)
+        # replaced for consistent errors
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_profile.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_profile.py [hunk 35]
```diff
@@ -414,9 +419,10 @@ def test_update_stewarded_orgs_for_apps_mall_steward_level_serializer_exception(
         data = {'display_name': 'Winston Smith', 'stewarded_organizations': False}
         response = self.client.put(url, data, format='json')
 
-        expected_data = {'stewarded_organizations': {'non_field_errors': ['Expected a list of items but got type "bool".']}}
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(response.data, expected_data)
+        # ConsistentErrors   expected_data = {'stewarded_organizations': {'non_field_errors': ['Expected a list of items but got type "bool".']}}
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_profile.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_review.py [hunk 36]
```diff
@@ -81,7 +82,9 @@ def test_create_review_not_found(self):
         url = '/api/listing/{0!s}/review/'.format(bread_basket_id)
         data = {'rate': 4, 'text': 'rutherford test review'}
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+        # ConsistentErrors  self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_review.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_review.py [hunk 37]
```diff
@@ -171,7 +174,8 @@ def test_delete_review(self):
         user = generic_model_access.get_profile('jones').user
         self.client.force_authenticate(user=user)
         response = self.client.delete(url, format='json')
-        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        # ConsistentErrors self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```
Comment (mannyrivera2010) [inline tests/ozpcenter_api/test_api_review.py]: Remove Comment?

File: tests/ozpcenter_api/test_api_notification.py [hunk 38]
```diff
@@ -495,9 +487,7 @@ def test_create_agency_notification_app_mall_steward_invalid_format(self):
         user = generic_model_access.get_profile('bigbrother').user
         self.client.force_authenticate(user=user)
         response = self.client.post(url, data, format='json')
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-
-        self.assertEqual(response.data['non_field_errors'], ['Valid Agency ID is required'])
+        self.assertEqual(response.data['error_code'], (ExceptionUnitTestHelper.validation_error('Valid Agency ID is required'))['error_code'])
```
Comment (cwilliams2017) [inline tests/ozpcenter_api/test_api_notification.py]: Remove line 490, and add below
```
self.assertEqual(response.data, ExceptionUnitTestHelper.validation_error('Valid Agency ID is required'))
```
you can compare all three fields at the same time

**Comment by mannyrivera2010** [comment]
:+1:

**Comment by rkenyon1969** [comment]
:+1:

**Pull request merged**